appsec/tests/integration/src/test/groovy/com/datadog/appsec/php/integration/CommonTests.groovy

```diff
@@ -222,6 +222,7 @@ trait CommonTests {
         assert span.metrics."_dd.appsec.enabled" == 1.0d
         assert span.metrics."_dd.appsec.waf.duration" > 0.0d
         assert span.meta."_dd.appsec.event_rules.version" != ''
+        assert span.meta."_dd.appsec.fp.http.endpoint" != ''
```

Comment:
Would it be possible to use here a regex to verify the format?

Reply:
Done 👌